packages/dci-opencrvs-bridge/src/opencrvs-to-dci.ts

```diff
@@ -195,7 +221,8 @@ export function searchResponseBuilder(
       page_number: pageNumber,
       page_size: pageSize,
       total_count: registrations.length
-    }
+    },
+    locale: 'en' // TODO: How do we figure this out?
```

Comment:
This would probably come from the request I guess. I changed the validation for this according to the specs here btw: https://github.com/opencrvs/dci-crvs-api/pull/37/files#diff-37451b2fb4ee7c252ba6efa0aa02772566ec8ffa6bc58ecd84511b287edccfa8R40